### ISSUE TITLE
ETA "Infinity"s

### ISSUE THREAD

**Issue by AxelTerizaki** [AndiDittrich/Node.CLI-Progress#21]
Hello :)

I have one minor problem with cli-progress so far.

I'm using it to display a download progress bar, but it sometimes is for really big files, and with some slow connections, sometimes "Infinity" is displayed instead of the time in the ETA variable.

Would it be possible to replace it by something else, like a -, ~, or the infinite symbol perhaps, when this happens? Having "Infinitys" sometimes bugs the progress bar into making new lines because it suddenly grows out of the number of characters allowed per line.

Thanks in advance :)

**Comment by AndiDittrich** [comment]
hi @AxelTerizaki ,

thanks for your report!
could you reproduce the issue with a simple testcase ? this would be great :)

btw, version 2.x of cli-progress has an option to disable line wrapping (no new lines, content is stripped to max line length)

best regards, Andi

**Comment by AxelTerizaki** [comment]
Hmmm, going to be difficult just like that but I'll think about it.

I suppose I could simulate a very slow transfer :) Can't promise to do it before next week though! It's not a big big issue so...

**Comment by AxelTerizaki** [comment]
Here's some code I just used : 

```
const _cliProgress = require('cli-progress');
import {promisify} from 'util';
const sleep = promisify(setTimeout);

async function updateBar() {
	bar1.update(1);
	await sleep(3600);	
}

// create a new progress bar instance and use shades_classic theme
const bar1 = new _cliProgress.Bar({}, _cliProgress.Presets.shades_classic);

// start the progress bar with a total value of 200 and start value of 0
bar1.start(99999999999999999999999999999999999999999999999999999999, 0);


// update the current value in your application..
Promise.resolve().then(function resolver() {
	return updateBar()
		.then(resolver);
});
```
It updates the bar after one hour, and since the bar's pretty big, it has a hard time calculating an ETA. I end up with that after 30 seconds or so : 

```
 ░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░ 0% | ETA: Infinitys | 1/1e+56
```

Hope it'll help :)

**Comment by AndiDittrich** [comment]
thanks!

i will take a look on it soon.

**Comment by AndiDittrich** [comment]
should be addressed in ac29f07
